--- .zshrc	2017-09-21 13:26:50.000000000 -0400
+++ .zshrc.mac	2017-09-21 16:30:10.000000000 -0400
@@ -17,6 +17,8 @@
 
 export GOPATH="${HOME}/.go"
 
+export PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
+
 export PATH="${PATH}:${HOME}/.gem/ruby/2.4.0/bin"
 #export PATH="$PATH:$HOME/.rvm/bin" # Add RVM to PATH for scripting
 #[[ -s "$HOME/.rvm/scripts/rvm" ]] && source "$HOME/.rvm/scripts/rvm" # Load RVM into a shell session *as a function*
