--- /Users/junwoo/.config/nvim/init.vim	2018-10-18 20:30:57.858295252 -0400
+++ init.vim	2018-08-26 20:17:24.819840000 -0400
@@ -93,8 +93,8 @@
 "                                                                     /____/
 
 " set python interpreter
-let g:python3_host_prog = '/usr/local/bin/python3'
-let g:python_host_prog = '/usr/local/bin/python2'
+let g:python3_host_prog = '/usr/bin/python3'
+let g:python_host_prog = '/usr/bin/python2'
 
 " set swap, backup and undo directory
 set directory=~/.vim/swap//
