diff --git a/neovim/init.vim b/Users/junwoo/.config/nvim/init.vim
index 0fe65b7..1488959 100644
--- a/neovim/init.vim
+++ b/Users/junwoo/.config/nvim/init.vim
@@ -92,8 +92,8 @@ call plug#end()
 "                                                                     /____/
 
 " set python interpreter
-let g:python3_host_prog = '/usr/bin/python3'
-let g:python_host_prog = '/usr/bin/python2'
+let g:python3_host_prog = '/usr/local/bin/python3'
+let g:python_host_prog = '/usr/local/bin/python2'
 
 " true color mode
 set termguicolors
@@ -108,8 +108,6 @@ set mouse=
 colorscheme onedark
 set background=dark
 let g:onedark_terminal_italics = 1
-hi Normal ctermbg=None guibg=None
-hi NonText ctermbg=None guibg=None
 
 " set line numbers
 set nonumber
@@ -121,7 +119,7 @@ set softtabstop=0
 set shiftwidth=4
 
 " set key combination timeouts
-set timeout=1
+set timeout
 set timeoutlen=500
 
 " update time
